profiler/src/ProfilerEngine/Datadog.Profiler.Native/FrameStore.cpp

```diff
@@ -1349,11 +1292,12 @@ PCCOR_SIGNATURE ParseElementType(IMetaDataImport* pMDImport,
             }
             if (SUCCEEDED(hr))
             {
-                size_t convertedChars;
-                wcstombs_s(&convertedChars, classname, sizeof(classname) / sizeof(char), zName, sizeof(zName) / sizeof(WCHAR));
+                StrAppend(buffer, shared::ToString(shared::WSTRING(zName)).c_str(), cchBuffer);
```

Comment:
there is a `shared::ToString` overload which converts `WCHAR` to `std::string`. This will avoid an allocation to `WSTRING`

Reply:
✅ ok